proc.go

```diff
@@ -79,9 +79,12 @@ var _ gioWindow = (*app.Window)(nil)
 // Proc runs the bound Setup function once before the event loop.
 // Proc then runs the bound Draw function once per event loop iteration.
 type Proc struct {
-	Setup Func
-	Draw  Func
-	Mouse Func
+	Setup       Func
+	Draw        Func
+	Mouse       Func
+	KeyPressed  KeyEventFunc
+	KeyTyped    KeyEventFunc
+	KeyReleased KeyEventFunc
```

Comment:
instead of "exploding" the `KeyCb` fields into 3 different fields in `Proc`, perhaps just add a `Key KeyCb` field to `Proc` ?